codegen/src/main/java/org/web3j/codegen/unit/gen/CompilerClassLoader.java

```diff
@@ -93,8 +92,7 @@
     }
 
     private String buildClassPath() {
-        final URL[] systemUrls = ((URLClassLoader) getClass().getClassLoader()).getURLs();
-        return buildClassPath(urls) + ':' + buildClassPath(systemUrls);
+        return buildClassPath(urls) + ':' + System.getProperty("java.class.path");
```

Comment:
This might need testing on Windows? From docs:

Path used to find directories and JAR archives containing class files. Elements of the class path are separated by a platform-specific character specified in the path.separator property.

platform-specific character

Reply:
ill test it on my windows machine now.

Reply:
This works on windows